src/body/body.rs

```diff
@@ -75,10 +75,19 @@ enum DelayEof {
     Eof(DelayEofUntil),
 }
 
-/// A sender half used with `Body::channel()`.
+/// A sender half created through [`Body::channel()`].
 ///
-/// Useful when wanting to stream chunks from another thread. See
-/// [`Body::channel`](Body::channel) for more.
+/// Useful when wanting to stream chunks from another thread.
+///
+/// ## Body Closing
+///
+/// Note that the request body will always be closed normally when the sender is dropped (meaning
+/// that the empty terminating chunk will be sent to the client). If you desire to close the
```

Comment:
```suggestion
/// that the empty terminating chunk will be sent to the remote). If you desire to close the
```

It could be server -> client, or client -> server.

Reply:
Good point!